x-pack/test_serverless/functional/test_suites/search/advanced_settings.ts

```diff
@@ -18,14 +18,10 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
   describe('Search advanced settings', function () {
     before(async () => {
-      await pageObjects.svlCommonPage.loginWithRole('viewer');
+      await pageObjects.svlCommonPage.loginWithPrivilegedRole();
```

Comment:
Why change the `viewer` role too, aren't we giving more permissions than necessary to the test?

Reply:
I thought this as well. Also, can we use the same role in the advanced settings tests in every serverless project type for consistency:
`x-pack/test_serverless/functional/test_suites/security/ftr/advanced_settings.ts`
`x-pack/test_serverless/functional/test_suites/observability/advanced_settings.ts`

Reply:
https://github.com/elastic/kibana/pull/188128/commits/e681a4e6ec5b0f05d05412dd9f1ffa59a013b54b